decidim-admin/app/views/decidim/admin/participatory_process_attachments/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h3><%= translated_attribute(@participatory_process_attachment.title) %></h3>
```

Comment:
In general terms, don't think these `show` pages provide much value. All the information shown here could probably be available in the `index` view. Just sayin'. Not asking to change it though.